src/languages/scala.js

```diff
@@ -151,13 +151,44 @@ export default function(hljs) {
     beginScope: { 2: "keyword", }
   };
 
+  // glob all non-whitespace characters as a "string"
+  // sourced from https://github.com/scala/docs.scala-lang/pull/2845
+  const DIRECTIVE_VALUE = {
+    className: 'string',
+    begin: /\S+/,
+  }
+
+  // glob all non-whitespace characters as a "type", so that we can highlight differently to values
+  // sourced from https://github.com/scala/docs.scala-lang/pull/2845
+  const DIRECTIVE_KEY = {
+    className: 'type',
+    begin: /\S+/,
+  }
+
+  // directives
+  // sourced from https://github.com/scala/docs.scala-lang/pull/2845
+  const USING_DIRECTIVE = hljs.COMMENT('//>', '\n', {
+    contains: [
```

Comment:
I think this would be simpler using a multi-mother for the first part, then contains for just the DIRECTIVE_VALUES... example of multi-match:

https://github.com/highlightjs/highlight.js/blob/main/src/languages/wren.js#L109

Reply:
Either starting from comment or perhaps starting from `using` if truly the whole line is still ALSO considered a comment...

Reply:
thanks for the tip! I used begin as a match instead